backend/package.json

```diff
@@ -1,28 +1,24 @@
 {
   "dependencies": {
-    "@mozilla/readability": "^0.4.2",
     "@nlpjs/lang-en": "^4.23.4",
-    "@types/isomorphic-fetch": "^0.0.36",
-    "cors": "^2.8.5",
-    "express": "^4.18.2",
-    "jsdom": "^20.0.2"
+    "aws-lambda": "^1.0.7",
+    "aws-sdk": "^2.1323.0"
   },
   "scripts": {
+    "build": "tsc",
     "start": "tsc && node dist/app.js",
```

Comment:
Is the `start` command still in use? If not, can we remove this?

Reply:
yes will remove